db/migrate/20191204112549_create_repository_number_value.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class CreateRepositoryNumberValue < ActiveRecord::Migration[6.0]
+  def change
+    create_table :repository_number_values do |t|
+      t.decimal :data, index: true
+      t.references :last_modified_by, index: true, foreign_key: { to_table: :users }
+      t.references :created_by, index: true, foreign_key: { to_table: :users }
+
+      t.timestamps
+    end
+
+    # add jsonb column for repository columns metadata
+    add_column :repository_columns, :metadata, :jsonb, default: {}, null: false
```

Comment:
Hmm.. this line is quite hidden here "inside" of table creatin migration. Maybe separate migration? Just thinking loudly